fxmanifest.lua

```diff
@@ -1,9 +1,7 @@
-games {'gta5'}
-
 fx_version 'cerulean'
 
 description 'Define zones of different shapes and test whether a point is inside or outside of the zone'
-version '2.6.1'
+version '2.6.0'
```

Comment:
Versions should not be downgraded, that's not how semver works. Patch version changes are for fixes that don't cause breaking changes. This does cause a breaking change (removing the config), which would normally entail a major version bump (3.x.x), but as mentioned in the other comment, we just shouldn't be removing that config. The ability to save zones both ways is a feature, and it def shouldn't be removed in a "fix this error" PR. Ideally, if this PR were just fixing the save error, the correct version bump would be `2.6.2`.